api.md

```diff
@@ -139,3 +148,15 @@ Methods:
 Custom Methods:
 
 - VerifySignature
+
+# Employer
```

Comment:
Let's exclude this as per https://github.com/Finch-API/finch-api-go/pull/21/commits/723b3f7bc37f6058603c475656b9f086154874ff#r1344028515